<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qlkh</class>
 <widget class="QMainWindow" name="qlkh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QUẢN LÝ KHÁCH HÀNG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;QUẢN LÝ KHÁCH HÀNG&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_timkiemkh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TÌM KIẾM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTenKhachHang">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>611</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>MÃ KHÁCH</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TÊN KHÁCH</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TUỔI</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>GIỚI TÍNH</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SỐ ĐIỆN THOẠI</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ĐỊA CHỈ</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_themkh">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>THÊM KHÁCH HÀNG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_suakh">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SỬA KHÁCH HÀNG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_xoakh">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XÓA KHÁCH HÀNG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quaylai2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>QUAY LẠI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>641</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../anhnen/R.jpg</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>btn_timkiemkh</zorder>
   <zorder>lineEditTenKhachHang</zorder>
   <zorder>tableWidget</zorder>
   <zorder>btn_themkh</zorder>
   <zorder>btn_suakh</zorder>
   <zorder>btn_xoakh</zorder>
   <zorder>btn_quaylai2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
